content/applications/hr/payroll.rst

```diff
@@ -59,25 +59,22 @@ are correctly entered.
 New work entry type
 ~~~~~~~~~~~~~~~~~~~
 
-To create a new work entry type, click the :guilabel:`Create` button. Enter the information on the
-form:
+To create a new :guilabel:`Work Entry Type`, click the :guilabel:`New` button, and enter the
+information on the form:
 
 - :guilabel:`Work Entry Type Name`: The name should be short and descriptive, such as `Sick Time` or
   `Public Holiday`.
-- :guilabel:`Code`: This code appears with the work entry type on timesheets and payslips. Since the
-  code is used in conjunction with the *Accounting* application, it is advised to check with the
-  accounting department for a code to use.
-- :guilabel:`Sequence`: The sequence determines the order that the work entry is computed in the
-  payslip list.
-- Check boxes: If any of the items in the list applies to the work entry, check off the box by
-  clicking it. If :guilabel:`Time Off` is checked off, a :guilabel:`Time Off Type` field appears.
-  This field has a drop-down menu to select the specific type of time off, or a new type of time off
-  can be entered.
-
-.. image:: payroll/new-work-entry.png
-   :align: center
-   :alt: New work entry type form.
-
+- :guilabel:`Payroll Code`: This code appears with the work entry type on timesheets and payslips.
+  Since the code is used in conjunction with the *Accounting* application, it is advised to check
+  with the accounting department for a code to use.
+- :guilabel:`DMFA code`: This code is used to identify DMFA entries on a corresponding DMFA report,
+  and is for Belgian-based companies only.
+- :guilabel:`External Code`: This code is used for exporting data to a third-party payroll service.
+  Refer to the third-party being used in order to determine the code to enter for the specific work
+  type entry.
+- :guilabel:`SDWorx code`: This code is only for companies that use SDWorx, a payroll service
+  provider.
```

Comment:
Correct - this would be used only when exporting the information in a spreadsheet, that you would then import to the third-party. It's just so the third-party can read/see the data, then know where to put whatever data in their app, not an actual integration. It's really a work-around = )